docs/topics/proc_deploying-application-to-standalone-rhel.adoc

```diff
@@ -0,0 +1,30 @@
+[id='deploying-application-to-standalone-rhel_{context}']
+= Deploying an application to standalone Red Hat Enterprise Linux
+
+You can deploy a {VertX} application to Red Hat Enterprise Linux.
+
+.Prerequisites
+
+* Red Hat Enterprise Linux 7
+* OpenJDK 8 installed
+* Maven 3.5 or later installed
+
+.Procedure
+
+. Download the {VertX} runtime from the link:https://maven.repository.redhat.com/[Red Hat Maven repository] to your RHEL 7 system.
```

Comment:
I'm fairly sure that's not how it works. Maven imports the Vert.x artifacts at build time, so it is my guess we only want to ask the user to make sure they have the RH repository configured in the POM file in the application.